stores/stripe/issuing.fga

```diff
@@ -0,0 +1,34 @@
+module issuing
```

Comment:
Looks like an odd module name. What would be the app component handling this called? "card-management""